api/blueprints/drbCollection.py

```diff
@@ -143,6 +143,38 @@ def collectionUpdate(uuid, user=None):
 
     return APIUtils.formatOPDS2Object(201, opdsFeed)
 
+@collection.route('/update/<uuid>', methods=['POST'])
+def collectionUpdate(uuid):
+    logger.info('Handling collection replacement request')
+
+    dbClient = DBClient(current_app.config['DB_CLIENT'])
+    dbClient.createSession()
+
+    title = request.args.get('title', None)
+    creator = request.args.get('creator', None)
+    description = request.args.get('description', None)
+
+    #Getting the collection the user wants to update
+    try:
+        collection = dbClient.fetchSingleCollection(uuid)
+    except NoResultFound:
+        errMsg = {'message': 'Unable to locate collection {}'.format(uuid)}
+        return APIUtils.formatResponseObject(404, 'fetchSingleCollection', errMsg)
+
+    collection.title = title
+    collection.creator = creator
+    collection.description = description
```

Comment:
one thing to consider: if the request doesn't contain one of these fields, we'll essentially be overwriting the given field to `None`. We may want to only update the fields that are present in the request object!